packages/components/src/components/hds/button/index.ts

```diff
@@ -6,6 +6,7 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 import type { HdsInteractiveSignature } from '../interactive/';
+import type { IconName } from '@hashicorp/flight-icons/svg';
```

Comment:
This needs to be updates too, to import the type from '@hashicorp/ember-flight-icons/components/flight-icon' as `FlightIconSignature`